composer.json

```diff
@@ -38,7 +38,8 @@
         "zumba/amplitude-php": "^1.0"
     },
     "conflict": {
-        "drupal/drupal": "*"
+        "drupal/drupal": "*",
+        "acquia/blt-behat": "<=1.0.0"
```

Comment:
@mikemadison13 I think this will prevent users from upgrading BLT before the plugin and breaking Behat integration... does it look right to you? Kind of hard to test.